CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
setup.py adds version to 0.16.0 so header for that version is needed here

Reply:
Tho I'd suggest let's do the release in a different PR

Reply:
Removed version bump from setup.py